### ISSUE TITLE
Extend the owner property

### ISSUE THREAD

**Issue by CSurieux** [OrchardCMS/OrchardCore#5529]
It would be good to set up a more advanced interface for this property than the 'simple user', a paradigm that would allow several users (a Team) to own a content  and thus be able to modify it under their own connection.
Ideally a Team should also support the roles.
I had implemented this in Orchard 1, at the cost of quite heavy modifications in the User processes, but I am not advanced enough to undertake it on Orchard Core ?

**Comment by deanmarcussen** [comment]
and also related https://github.com/OrchardCMS/OrchardCore/issues/3492 with some good comments.

**Comment by pgy866** [comment]
yes,It's related. What if you add a global database and a site database? This also enables tenant nesting for unlimited levels of tenants

**Comment by CSurieux** [comment]
I have already read this interesting thread, but may be username is related to user, not to owner.
It is the actual implementation which use this key to link owner and user.
May be owner should be something more evolved as (cat of, key of) ?

**Comment by sebastienros** [comment]
You can create custom parts that would have more data if necessary.
